tests/configs/test_main.py

```diff
@@ -0,0 +1,46 @@
+import os
+from pathlib import Path
+
+import pytest
+
+from gnatss.configs.main import Configuration
+
+
+def test_env_configuration_main():
+    os.environ.setdefault("GNATSS_SITE_ID", "test_site")
+    with pytest.warns(
+        UserWarning,
+        match=(
+            "Configuration file `config.yaml` not found. "
+            "Will attempt to retrieve configuration from environment variables."
+        ),
+    ):
+        config = Configuration()
+
+    assert config.site_id == "test_site"
+
+
+def test_env_main_posfilter():
+    # Create test file
+    test_file = Path(__file__).parent / "test_data.csv"
+    test_file.touch()
+
+    # Set environment variables
+    os.environ.setdefault("GNATSS_SITE_ID", "test_site")
```

Comment:
Can we move this to `@pytest.fixture` since it is used in the previous function as well?